spec/features/cases/overturned_sar/create_case_spec.rb

```diff
@@ -45,6 +45,50 @@
                       type: :text
                     )
 
+    expect(form).not_to have_flag_for_disclosure_specialists
+
+    click_button 'Create case'
+
+    # Browse Business Group
+    assignments_new_page.choose_business_group(responding_team.business_group)
+
+    # Select Business Unit
+    assignments_new_page.choose_business_unit(responding_team)
+  end
+
+  scenario 'creating an OVT FOI case', js: true do
+    ico_case = create(:closed_ico_foi_case)
+
+    # Move this to a user journey test once displaying works.
```

Comment:
I don't think this comment is relevant anymore, is it?

Reply:
The comment is relevant - but the test isn't - it is covered in the e2e user journey test `spec/features/user_journeys/overturned_foi/ovt_foi_non_trigger_e2e_spec.rb`.  Have removed this spec.